package.json

```diff
@@ -67,6 +67,7 @@
     ]
   },
   "dependencies": {
+    "enzyme": "^3.1.0",
```

Comment:
✂️ - also test libs should generally be in `devDependencies` instead of `dependencies`

Reply:
@jxnblk this is never used. @Degron did you remove it?